app/views/govuk_publishing_components/components/layout_header/_search.html.erb

```diff
@@ -1,8 +1,17 @@
-<button class="search-toggle js-header-toggle" data-search-toggle-for="search">Show search</button>
-<form id="search" class="gem-c-layout-header__search-form govuk-clearfix" action="/search" method="get" role="search">
+<button class="search-toggle js-header-toggle" data-search-toggle-for="search">
+  Show search
```

Comment:
Another translation request for this and line 12.

Reply:
Yep, I should've done that. Fixed!